---

title: "Hang starting Terracotta hibernate cache when statistics DB not opened"
layout: issue
tags: 
permalink: /browse/CDV-1300

issue_key: CDV-1300
issue_numeric_sort_key: 1300
issuetype: "Bug"
project: "Community Development (Terracotta Server)"
project_key: "CDV"
status: "Closed"
resolution: "Fixed"
priority: "2 Major"
components: ""
labels: 
assignee: "kkannaiy"
reporter: "teck"
votes:  0
watchers: 0

created: "2009-06-22T16:14:57.000-0400"
updated: "2013-02-12T14:02:13.000-0500"
resolved: "2009-06-25T12:04:20.000-0400"

---



{% raw %}


## Attachments

* <em>teck</em> (27.000 k, text/plain) [dump.txt](/attachments/CDV/CDV-1300/dump.txt)




## Description

<div markdown="1" class="description">

I accidentally configured my two L1s to use the same path for the statistics store. The core L1 client start fine (albeit with a big warning about not being able to open the stats DB). Unfortunately something about the hibernate/SRA stuff hangs. It seems that it waits forever for things to become available (which is never going to happen in this case) 

Attached is the thread dump


</div>

## Comments


{:.comment-heading}
### **Geert Bevin** <span class="date">2009-06-25</span>

<div markdown="1" class="comment">

12967 in 3.1 branch

</div>


{:.comment-heading}
### **Kalai Kannaiyan** <span class="date">2009-08-13</span>

<div markdown="1" class="comment">

Verified with  Terracotta 3.1.0-SNAPSHOT, as of 20090812-140815 (Revision 13403 by cruise@su10mo4 from 3.1)  with h2lcperf app,
configured two L1's using the same path of statistics db, the clients and server are working fine.


</div>



{% endraw %}
